packages/boilerplates/ExpectedBoilerplateError.ts

```diff
@@ -0,0 +1,10 @@
+/*---------------------------------------------------------------------------------------------
+*  Copyright (c) Dolittle. All rights reserved.
+*  Licensed under the MIT License. See LICENSE in the project root for license information.
+*--------------------------------------------------------------------------------------------*/
+export class ExpectedBoilerplateError extends Error {
```

Comment:
Postfixing not needed. 
If this is an exception, it should be more like `BoilerplateIsExpected`- we have guidance on this for C#, which we need to generalise: https://dolittle.io/contributing/guidelines/csharp_coding_styles/